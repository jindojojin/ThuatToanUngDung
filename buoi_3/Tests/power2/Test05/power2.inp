100
12 42 5 16 43 54 33 4 3 17 7 18 4 5 1 4 5 3 4 3 19 4 5 111 4 5 3 4 3 11 7 111 11 5 11 4 5 3 4 3 1 4 5 1 4 5 3 4 3 1 7 1 4 5 1 4 5 3 4 3 1 4 5 1 2048 5 3 4 3 1 7 1 4 5 1 4 5 3 4 3 1 4 5 1024 4 5 3 4 3 1 7 1 4 512 1 111 5 3 222 3
