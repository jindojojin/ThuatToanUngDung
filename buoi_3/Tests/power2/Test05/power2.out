1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 4 4 4 4 4 4 4 4 4 4 4 4 4 4 4 4 4 4 4 4 4 4 4 4 16 512 1024 2048
